example-wasm/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
What does this do?

Reply:
It makes it possible to [import](https://www.typescriptlang.org/docs/handbook/modules.html#ambient-modules) the declared modules in `realm-js/node_modules/react-scripts/lib/react-app.d.ts` (which in turn also references the declaration files of `@types/node`, `@types/react`, and `@types/react-dom`).

Reply:
Cool, thanks for the explanation